README.md

```diff
@@ -1,5 +1,71 @@
 # molgenis-ui-filter
 
+## Filter Container
+
+The filter container takes a descriptive input array to create a set of filters. It will also require a 2 way bound output object to receive the filters state
+
+```
+<FilterContainer
+  v-model="selections"
+  :filters="filters"
+/>
+```
+
+### Filter input object specifications
+
+```
+filters: [ {
+    name: 'search',
+    label: 'Search',
+    description: 'search by name',
+    collapsed: true,
+    placeholder: 'test',
+    type: SearchFilterVueObject
+}, {
+    name: 'country',
+    label: 'Countries',
+    collapsed: false,
+    options: [{ value: 'nl', text: 'Nederland' }, { value: 'de', text: 'Duitsland' }, { value: 'be', text: 'België' }],
+    type: CheckboxVueObject
+}]
+```
+
+Every filter will have the following global properties:
+
+| name        | required | default value | description | 
+|-------------|----------|---------------|-------------|
+| name        | true     | nvt           | string: name and id of the filter |
+| label       | false    |               | string: Name of the filter as used by the UI |
+| description | false    |               | Descriptive text that is used by the UI |
+| collapsed   | false    | true          | boolean: Open/Closed state of the card that holds the filter |
+| type        | true     | nvt           | Vue Object that renders the filter |
+
+The `type` field takes a VUE object that renders a filter. Note that every object can have additional filter properties
+
+### Additional filter properties
+#### CheckboxFilter 
+`options`: Describe the checkboxes. The value property is used as key and the text property is used as label by the UI
+```
+[{ value: 'value1', text: 'Checkbox 1' }, { value: 'value2', text: 'Checkbox 2' }]
```

Comment:
what a about an id or key for when two options have the same value but a differ in text